sdk/sync/subscription.go

```diff
@@ -52,8 +35,7 @@ func (s *subscription) process() {
 
 	startSeq, err := s.client.XLen(key).Result()
 	if err != nil {
-		s.connCh <- -1
-		s.result = fmt.Errorf("failed to fetch current length of stream: %w", err)
+		s.w.re.SLogger().Errorf("failed to fetch current length of stream: %w", err)
```

Comment:
Before: We'd learn about errors from `Close` and log them.
Now: We just log the errors.

The "correct" way to do this is probably to return an error channel from `Subscribe`. Is that worth it?